dapp-core/src/main/kotlin/jp/co/soramitsu/dapp/cache/DefaultCacheManager.kt

```diff
@@ -0,0 +1,36 @@
+package jp.co.soramitsu.dapp.cache
+
+import jp.co.soramitsu.dapp.helper.CacheManager
+import org.springframework.stereotype.Component
+
+@Component
+class DefaultCacheManager : CacheManager {
+
+    private val cache = HashMap<String, Any?>()
+
+    override fun put(key: String?, value: Any?): Any? {
+        checkArgument(key)
+        return cache.put(key!!, value)
+    }
+
+    override fun get(key: String?): Any? {
+        checkArgument(key)
+        return cache[key]
+    }
+
+    override fun remove(key: String?): Any? {
+        checkArgument(key)
+        return cache.remove(key)
+    }
+
+    override fun contains(key: String?): Boolean {
+        checkArgument(key)
+        return cache.contains(key)
+    }
+
+    private fun checkArgument(key: String?) {
```

Comment:
just make key `String` with no question mark.

Reply:
Methods signatures come from Java interface so I need to add `!!` in that case, should I do it?

Reply:
If it compiles you may not use a question mark.

Reply:
Okay, made it just `String`